### ISSUE TITLE
Open Sans doesn't support inverted e "ə" character

### ISSUE THREAD

**Issue by jhell** [google/fonts#1602]
Open Sans doesn't support the[ latin character "ə"](https://en.wikipedia.org/wiki/%C6%8F). This character is used in the Azerbaijani language.
Here is an exemple of how it looks in [Google Font](https://fonts.google.com/specimen/Open+Sans?selection.family=Open+Sans) : 
![az](https://user-images.githubusercontent.com/1507165/41842383-ef052f32-786a-11e8-9337-b29458f6516e.PNG)

**Comment by mostalt** [comment]
Hello.
What progress on this issue? Bug still exists.

**Comment by applecuckoo** [comment]
The v3 builds in the [Open Sans repo](https://github.com/googlefonts/opensans/tree/main/fonts/variable) seems to have this bug fixed, maybe this issue can be closed?
![inverted_e](https://user-images.githubusercontent.com/113647417/191874261-c136c988-8181-424e-a2fe-5b9237a31234.png)

**Comment by fitojb** [comment]
Yep, the character is there now in the font, although its spacing in the italic styles is a little funky. But that is another issue.
